### PR TITLE
Cluster shared data

### PR DESCRIPTION
**Pull request by naveed050**

### CODE REVIEW

File: src/main/scala/com/imaginea/activegrid/core/models/SharedSessionCache.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    LWWMapKey(entryKey)
+
+  def receive: PartialFunction[Any, Unit] = {
+    case PutSession(key, value) =>
+      logger.info(s"inserting terminal session in shared map with key: $key")
+      replicator ! Replicator.Update(dataKey(key), LWWMap(), writeMajority)(_ + (key -> value))
+    case RemoveSession(key) =>
+      logger.info(s"removing terminal session in shared map with key: $key")
+      replicator ! Replicator.Update(dataKey(key), LWWMap(), writeMajority)(_ - key)
+    case GetSession(key) =>
```
Comment (eklavya) [inline src/main/scala/com/imaginea/activegrid/core/models/SharedSessionCache.scala]: We discussed that it should be changed.

Comment (naveed050) [reply src/main/scala/com/imaginea/activegrid/core/models/SharedSessionCache.scala]: We discussed that Siva should use ask in his code. But I don't remember anything to be changed here.
Could you tell me what needs to be changed and I will change it.

Comment (eklavya) [reply src/main/scala/com/imaginea/activegrid/core/models/SharedSessionCache.scala]: Yes, Siva needed to change it. How come it's in your PR?

Comment (naveed050) [reply src/main/scala/com/imaginea/activegrid/core/models/SharedSessionCache.scala]: Because I wrote this. This was same when I showed you last time.

**Pull request merged**